devicetypes/smartthings/arrival-sensor-ha.src/arrival-sensor-ha.groovy

```diff
@@ -88,27 +88,33 @@ private handleReportAttributeMessage(String description) {
     }
 }
 
-private handleBatteryEvent(rawValue) {
-    def linkText = getLinkText(device)
-
+/**
+ * Create battery event from reported battery voltage.
+ *
+ * @param volts Battery voltage in .1V increments
+ */
+private handleBatteryEvent(volts) {
     def eventMap = [
         name: 'battery',
         value: '--'
```

Comment:
That might make more sense. This along with the map work that you've done for battery is something that we should standardize for other battery devices